atmo/decorators.py

```diff
@@ -67,3 +67,29 @@ def change_permission_required(model, **params):
 
 def delete_permission_required(model, **params):
     return permission_required(full_perm(model, 'delete'), model, **params)
+
+
+def modified_date(view_func):
```

Comment:
This has a smell of ETag headers. I haven't read and understood everything here yet but I'm curious if we might have used ETags or `Last-Modified` header.